src/machines/java/com/enderio/machines/common/blockentity/task/SpawnTask.java

```diff
@@ -96,6 +108,34 @@ public boolean isAreaClear() {
         }
         return true;
     }
+    
+    private void loadSoulData(Optional<ResourceLocation> rl) {
+        if (rl.isEmpty()) {
+            blockEntity.setReason(PoweredSpawnerBlockEntity.SpawnerBlockedReason.UNKOWN_MOB);
+            return;
+        }
+        Optional<EntityType<?>> optionalEntity = Registry.ENTITY_TYPE.getOptional(rl.get());
+        if (optionalEntity.isEmpty() || !Registry.ENTITY_TYPE.getKey(optionalEntity.get()).equals(rl.get())) {
+            blockEntity.setReason(PoweredSpawnerBlockEntity.SpawnerBlockedReason.UNKOWN_MOB);
+            return;
+        }
+        if (optionalEntity.get().is(MachineTags.EntityTypes.SPAWNER_BLACKLIST)) {
+            return;
+        }
+        Optional<SpawnerSoul.SoulData> opData = SpawnerSoul.SPAWNER.matches(rl.get());
+        if (opData.isEmpty()) { //Fallback
+            this.entityType = optionalEntity.get();
+            this.energyCost = 4000;
+            if (entityType.create(this.blockEntity.getLevel()) instanceof LivingEntity entity) { //Are we 100% guaranteed this is a living entity?
```

Comment:
I would argue that only living entities should be targetable. If there is any kind of exception discovered later, we can go for adding in additional cases.

Reply:
Yeah I agree